activities.json

```diff
@@ -142,6 +142,17 @@
     "title": "Feature Policy",
     "url": "https://wicg.github.io/feature-policy"
   },
+  {
+    "ciuName": null,
+    "description": "This document defines a set of Fetch metadata request headers that aim to provide servers with enough information to make a priori decisions about whether or not to service a request based on the way it was made, and the context in which it will be used.",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1508292",
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "This gives servers useful context about requests that can be used to mitigate various security issues. There are some thorny issues around embed/object that made agreeing on a design more complicated than expected.",
```

Comment:
I would phrase this as "There are aspects of the design that we feel are quite complicated, [...]" and "but these are unavoidable" or "and we would like to see more work to simplify over time"

Reply:
I reworded it a bit, more reasonable now?